affine.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def f(A, B, X):
```

Comment:
`f` is used here https://github.com/media-comp/2022-matmul/blob/main/simple_matmul.py#L4
So can you change the name of the function?

Reply:
Thank you. I changed the name of the function to affine for now.